src/main/java/projet/projetstage02/controller/RootController.java

```diff
@@ -1036,4 +1037,34 @@ public ResponseEntity<InterviewOutDTO> StudentSelectDate(@RequestBody InterviewS
             return ResponseEntity.status(NOT_FOUND).build();
         }
     }
+
+    @PostMapping("/evaluateStudent/{token}")
```

Comment:
Put the token in the DTO we did it with multiple DTOs. So let try to keep it uniform.